src/main/java/racing/view/ResultView.java

```diff
@@ -1,24 +1,38 @@
 package racing.view;
 
+import racing.domain.Judge;
+import racing.domain.RacingCars;
 import racing.domain.RacingCar;
 
-import java.util.List;
-
 public class ResultView {
     private static final String HYPHEN = "-";
+    private static final String COMMA = ",";
     private static final String LINE_BREAK = "\n";
 
     public static void printGameResultTitle() {
         System.out.println(LINE_BREAK + "실행 결과");
     }
 
-    public static void printCarCurrentPosition(List<RacingCar> racingCars) {
-        racingCars.stream()
-                .map(car -> positionToString(car.getPosition()))
+    public static void printCarsCurrentPosition(RacingCars racingCars) {
+        racingCars.getRacingCarList()
+                .stream()
+                .map(ResultView::toCurrentPositionString)
                 .forEach(System.out::println);
         System.out.print(LINE_BREAK);
     }
 
+    public static void printWinners(RacingCars racingCars) {
+        System.out.println(joinWinnerNames(racingCars) + " 가 최종 우승했습니다.");
+    }
+
+    private static String joinWinnerNames(RacingCars racingCars) {
+        return String.join(COMMA, Judge.selectWinners(racingCars));
```

Comment:
```suggestion
        return String.join(COMMA, racingCars.maxPositionNames());
```

상태를 가진 객체에게 물어보는건 어떨까요?

Reply:
우승자들의 이름을 추출하는 로직 또한 RacingCars 내부로 이동시켰습니다.
`Judge` 클래스는 외부에서 다른 객체들의 상태를 참조하여 계산했었는데,
각 상태를 가진 객체에게 책임을 부여하니 이 클래스가 불필요하게 되어 삭제하였습니다!